hub_sdk/config.py

```diff
@@ -6,6 +6,6 @@
     'ULTRALYTICS_FIREBASE_AUTH_URL',
     'http://localhost:9099/identitytoolkit.googleapis.com/v1/accounts:signInWithPassword?key=AIzaSyDlTep-ubgWoafviJJneFL35raoJjWFnOw'
 )
-HUB_FUNCTIONS_ROOT = f'{HUB_API_ROOT}'
+HUB_FUNCTIONS_ROOT = f'{HUB_API_ROOT}/rest'
```

Comment:
@hassaanfarooq01 This looks wrong? We don't want to direct to the Cloud Functions.